book/python/api.md

```diff
@@ -229,21 +229,21 @@ The TCP Source Decoder is responsible for two tasks:
 1. Telling Wallaroo _how many bytes to read_ from its input connection.
 2. Converting those bytes into an object that the rest of the application can process.
 
-Your function must be decorated with `@decoder(header_length, fmt)`.
+Your function must be decorated with `@wallaroo.decoder(header_length=header_length, length_fmt="length_fmt")`.
```

Comment:
`length_fmt` doesn't take a name string here. It takes an argument that is passed to `struct.unpack` following https://docs.python.org/2/library/struct.html#format-strings

Reply:
That's described in the `#### `length_fmt` section below, but the example probably shouldn't use an invalid argument either?

Reply:
any thoughts on what you'd propose as an optimal replacement to `"length_fmt"`?

Reply:
I'm tempted to use real values from our examples, like `@wallaroo.decoder(header_length=4, length_fmt=">I")`